### ISSUE TITLE
MMenu works great on all browsers except IE9. Is there a certain version that works in ie9? 

### ISSUE THREAD

**Issue by toniGood1** [FrDH/jQuery.mmenu#513]

**Comment by FrDH** [comment]
Is it not working great on IE9 or is it not working at all on IE9?

The latest version should work on IE9 and up.
However, IE9 has an odd issue that prevents the submenus from closing. That issue was fixed by immediately hiding the panels (without an animation) on IE9 only.

**Comment by toniGood1** [comment]
Thanks for the response. But it's not working at all in IE9. The examples of the site doesn't work on IE9 for me either. Can you check, perhaps its just me. http://mmenu.frebsite.nl/examples.html

**Comment by FrDH** [comment]
I switched to using Gulp in version 5.6, ever since the CSS is automaticllay prefixed to support the latest 2 versions of browsers.
You could try going back to [version 5.5](https://github.com/FrDH/jQuery.mmenu/releases/tag/v5.5.3)
Let me know if that version does work.

**Comment by toniGood1** [comment]
Thanks for the suggestion- I will give this a try.

**Comment by FrDH** [comment]
For the record, I've expanded browser support for the gulp-autoprefixer in the next version so it should add prefixes for older version sof IE as well.

**Comment by FrDH** [comment]
In version 5.6.2 I added vendor-prefixes for older browsers, I think it should also include IE9 now.

**Comment by toniGood1** [comment]
I'm using grunt for compiling my sass and none of the versions were working because it was being auto prefixed including 5.6.2. So what I did was - I tried 5.6.2 css version without compiling, but still not working in IE9. IE10-11 are working just fine though.  

I am however able to get 4.6.3 css version to work in IE9 without compiling with grunt, but I'm using it with 5.5.3 js because of options. Kind of a hack right now and perhaps when I get more time, I can look further into the css to pull what I need to from 4.6.3.
